### PR TITLE
Sign EnvDTE.Host

### PR DESCRIPTION
**Pull request by kirillgla**

### CODE REVIEW

**Review by denis417 — APPROVED** [review]
lgtm

File: EnvDTE.Host/EnvDTE.Host.csproj [hunk 1]
```diff
@@ -5,6 +5,8 @@
         <AssemblyName>JetBrains.EnvDTE.Host</AssemblyName>
         <RootNamespace>JetBrains.EnvDTE.Host</RootNamespace>
         <IsPackable>false</IsPackable>
+        <SignAssembly>true</SignAssembly>
+        <AssemblyOriginatorKeyFile>..\Subplatform.snk</AssemblyOriginatorKeyFile>
```
Comment (ForNeVeR) [inline EnvDTE.Host/EnvDTE.Host.csproj:9]: Won't this break the open-source build, folks? I can't see `Subplatform.snk` in this repo.

Comment (denis417) [reply EnvDTE.Host/EnvDTE.Host.csproj:9]: As I can see, it was added in this Pull Request

Comment (ForNeVeR) [reply EnvDTE.Host/EnvDTE.Host.csproj:9]: Ah, sorry, my bad. That's correct.

**Review by ForNeVeR — APPROVED** [review]

**Pull request merged**